core/pom.xml

```diff
@@ -11,13 +11,24 @@
 
   <artifactId>wrangler-core</artifactId>
 
+
   <dependencies>
     <dependency>
+      <groupId>co.cask.wrangler</groupId>
+      <artifactId>wrangler-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
       <version>${httpclient.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.11</version>
```

Comment:
Is it OK if you downgrade the version to 0.9.10? I tried using 0.9.11 version, and it gave method not found exception because of Guava version conflict, when I ran an unit test that create Reflections. 0.9.10 version works for me.